### ISSUE TITLE
Wrong S0 metering, only bad pulses

### ISSUE THREAD

**Issue by Alkazar-DH** [heishamon/HeishaMon#314]
Today I discovered a false power value in my smart home system, I checked Heishamon and found a lot of bad values in the console.
![image](https://user-images.githubusercontent.com/75751915/214231449-b3b2604f-8b38-47d0-9103-2400312bd1d7.png)
I using Heishamon V3.0 and it worked perfectly for around a half year.
Any suggestions how to solve this problem?

I did some heishamon reboots and finally i did a complete switch off by circuit breaker, to power down the energy meter as well. It looks now i little bit better, but still some bad pulses:
![image](https://user-images.githubusercontent.com/75751915/214236249-fe536fc5-a411-4914-99d0-c050205a63e9.png)

**Comment by IgorYbema** [comment]
This is because the line is picking up noise from around. Please use better cabling (shielded signal cable) and maybe add some ferrite beads to the signal line.

**Comment by jacco-hass** [comment]
I've also got this issue, even when using STP cabling and ferrite beads. I checked the pulses from my kWh-meter with an oscilloscope, which looks good (exactly 80ms pulse):
![image_2023-12-09_144259293](https://github.com/Egyras/HeishaMon/assets/62347605/7891394e-5a55-4db3-9fbd-7c844f8e9e88)

Any ideas what might be the issue? At first I suspected a dirty ground of the Heishamon board, but I tested different types of power supplies and the issue remains.
